tutorial/practicals/LFRic/building_code/4_psydata/Makefile

```diff
@@ -41,5 +41,21 @@ default:
 	@echo "Please use one of the transformation-specific Makefiles, e.g."
 	@echo "make -f Makefile.extract"
 
+# Target for testing the tutorials with compilation (use only one PSyData wrapper)
+compile:
+	make -f solutions/Makefile.nan_all
+
+
+# Target for CI, only do the transformation (use all transformation scripts)
+test:
+	make clean
```

Comment:
When running `make clean` and `make allclean` I noticed that neither of these targets removes the created algorithm `time_evolution_alg_mod.f90`. Please add this to the `clean` targets.

Reply:
Oh yes, it used an empty variable - fixed.

Reply:
Definitely fixed 😄